test/test_jit.py

```diff
@@ -1688,6 +1688,20 @@ def doit(x, y):
         for node in g.nodes():
             self.assertTrue(g2.findNode(node.kind()) is not None)
 
+    def test_permute_inputs_binding(self):
+        g = torch._C.Graph()
```

Comment:
looks like this is failing - maybe try by using a graph from a function ?

```
@torch.jit.script
def foo(i, j, k):
     pass

g = foo.graph
```

Reply:
my bad i should've test this locally before i push. thanks for the trip